common/common.mk

```diff
@@ -3,8 +3,8 @@
 CXX := clang++-10
 STDCXX ?= c++17
 CFLAGS := $(CFLAGS) -DGLSL_VERSION='"330 core"'
-DBG_FLAGS ?= -Og -g -Wall -Wextra -Werror
-OPT_FLAGS ?= -O3 -DNDEBUG -Wall -Wextra -Werror
+DBG_FLAGS ?= -Og -g -Wall -Wextra -Werror -Wno-missing-field-initializers
```

Comment:
`common/gl_util/lib/stb_image_write.h` initializes structs with { 0 }, which results in the missing-field-initializers warning.
If we don't want to put too many flags here, I can add each structure's initial value.

Reply:
It's better to use `#pragma GCC diagnostic` around the import site, so the warning is active for the rest of the code. See [this SO post](https://stackoverflow.com/q/3378560/1078199).

Reply:
Nice solution for external dependencies

Reply:
Removed the flags in common.mk and added `#pragma GCC diagnostic` in `offload_data/plugin.cpp`